tools/ReleaseTools.psm1

```diff
@@ -354,6 +372,8 @@ function New-DraftRelease {
         Pop-Location
     }
 
-    Get-GitHubRepository -OwnerName PowerShell -RepositoryName $RepositoryName |
-        New-GitHubRelease @ReleaseParams
+    $Repo = Get-GitHubRepository -OwnerName PowerShell -RepositoryName $RepositoryName
+    $Release = $Repo | New-GitHubRelease @ReleaseParams
+
+    $Assets | ForEach-Object { $Release | New-GitHubReleaseAsset -Path $_ }
```

Comment:
@rjmholt Is this how you would do it all pipeline-y? Or do I need to set this up with begin/process/end?

Reply:
It looks like [`New-GitHubReleaseAsset` takes the value of `-Path` from the pipeline](https://github.com/microsoft/PowerShellForGitHub/blob/0ebf9e8b8d0f4fa361996674dbca804da36bace7/GitHubReleases.ps1#L1024-L1030), so I'd write this like:

```powershell
$Assets | New-GitHubReleaseAsset -OwnerName PowerShell -RepositoryName $RepositoryName -Release $Release.id
```

Reply:
Can we send `$Repo` and `$Assets` at the same time?